### PR TITLE
Issue #1931: Layouts: empty .dropbutton-* <divs>

### PR DESCRIPTION
**Pull request by klonos**
Fixes https://github.com/backdrop/backdrop-issues/issues/1931

Replaces https://github.com/backdrop/backdrop/pull/1416

### CODE REVIEW

**Comment by Gormartsen** [comment]
Issue with deploy phaze. Need to close , wait 10 sec, and open PR again. Should be ok

**Comment by klonos** [comment]
...closing PR.

**Comment by klonos** [comment]
...reopening PR.

File: core/modules/layout/layout.admin.inc [hunk 1]
```diff
@@ -29,20 +29,18 @@ function layout_list_page() {
     $row = array();
     // Default layouts have a null path.
     if ($path === '') {
-      $row[] = array('data' => t('Default layouts (used on all paths without a specific layout)'));
-      // Empty columns are intentional so that they may be hidden with
-      // priority-low class on the $header array.
-      $row[] = '';
-      $row[] = '';
+      $row[] = array('data' => t('Default layouts (used on all paths without a specific layout)'), 'colspan' => '3');
```
Comment (klonos) [inline core/modules/layout/layout.admin.inc:32]: ...turns out that my method works only when all columns are visible. The `colspan` value needs to dynamically change (JS?) to `2` as one column gets hidden when the viewport gets smaller and then needs to be entirely removed when the only columns shown are the "Path and layout" and "Operations".

Beyond my skills, so anybody that wants to pick this up, feel free.

**Comment by Gormartsen** [comment]
Interesting. I checked deploy log and see that it wasn't able to install it this time.
I will rewrite borg_qa module to wait until database and domain created.

**Comment by klonos** [comment]
...yeah, I was about to say that the sandbox throws the "Choose language" page.

**Comment by quicksketch** [comment]
I cherry-picked out the first commit into 1.x and 1.4.x. We can try to solve the table-cell issue separately, that might take a more substantial effort.

**Pull request closed without merging**